src/WhatToCook.Application/DataTransferObjects/Requests/RecipeRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace WhatToCook.WebApp.DataTransferObject.Requests
```

Comment:
Inconsistent namespace, should be `WhatToCook.Application.DataTransferObjects.Requests`

Reply:
done, will be in the next commit